language-support/ts/codegen/src/TsCodeGenMain.hs

```diff
@@ -1021,24 +1025,24 @@ packageJsonDependencies (Scope scope) dependencies = object $
     ] ++
     [ Aeson.fromText (scope <> "/" <> pkgName) .= ("file:../" <> pkgName) | Dependency pkgName <- dependencies ]
 
-packageJsonPeerDependencies:: SdkVersion ->  Value
-packageJsonPeerDependencies sdkVersion = object
-    [ "@daml/types" .= versionToText sdkVersion
-    , "@daml/ledger" .= versionToText sdkVersion
+packageJsonPeerDependencies:: ReleaseVersion ->  Value
+packageJsonPeerDependencies releaseVersion = object
+    [ "@daml/types" .= versionToText releaseVersion
+    , "@daml/ledger" .= versionToText releaseVersion
```

Comment:
I assume you checked we publish these under the release version for split release?

Reply:
@garyverhaegen-da I assumed these would be published on a per release basis, but I assume I could be wrong because they exist in the `daml` repo.

Reply:
fixed to use sdkVersion in codegen